### PR TITLE
feat: added track changes

### PR DESCRIPTION
**Pull request by shnsumit**
- [ ] https://github.com/AcclaroInc/pm-craft-translations/issues/359

### CODE REVIEW

**Comment by shnsumit** [comment]
@sidedwards  Latest changes has been pushed, you can review it now whenever get time.

File: src/elements/Order.php [hunk 1]
```diff
@@ -181,12 +181,32 @@ protected static function defineSources(string $context = null): array
...
                 'key' => 'pending',
                 'label' => Translations::$plugin->translator->translate('app', 'Pending'),
                 'criteria' => [
                     'status' => [
-                        Constants::ORDER_STATUS_NEW
+                        Constants::ORDER_STATUS_PENDING
+                    ]
+                ],
+                'defaultSort' => ['dateOrdered', 'desc']
+            ],
```
Comment (sidedwards) [inline src/elements/Order.php:203]: @shnsumit, should we switch the order of this?

image

File: src/migrations/Install.php [hunk 2]
```diff
@@ -123,7 +123,7 @@ protected function createTables()
                     'ownerId'           => $this->integer()->notNull(),
                     'sourceSite'        => $this->integer()->notNull(),
                     'targetSites'       => $this->string(1020)->notNull()->defaultValue(''),
-                    'status'            => $this->enum('status', Constants::ORDER_STATUSES)->defaultValue(Constants::ORDER_STATUS_NEW),
+                    'status'            => $this->enum('status', Constants::ORDER_STATUSES)->defaultValue(Constants::ORDER_STATUS_PENDING),
```
Comment (sidedwards) [inline src/migrations/Install.php:126]: @shnsumit, we should also include the `ORDER_STATUS_NEW` in `Install.php`, right?

File: src/services/job/ImportFiles.php [hunk 3]
```diff
@@ -59,23 +59,15 @@ protected function defaultDescription()
...
 
-        if (empty($this->fileNames)) {
-            $this->fileNames = $fileNames;
-        }
+        $this->fileNames = $this->fileNames ?: $fileNames;
 
-        if (empty($this->disacradElements)) {
-            $this->disacradElements = $disacradElements;
-        }
+        $this->disacradElements = $this->disacradElements ?: $disacradElements;
```
Comment (sidedwards) [inline src/services/job/ImportFiles.php:68]: @shnsumit, was this variable supposed to be `disregardElements`?

**Review by sidedwards — CHANGES_REQUESTED** [review]
@shnsumit, please let me know your thoughts on the below feedback.

File: src/migrations/Install.php (hunk 2 above)
Comment (shnsumit) [reply src/migrations/Install.php:126]: @sidedwards  All new statuses were added to constant so it's going to add from there.

image

**Pull request merged**